src/views/certifications/views/[slug]/index.tsx

```diff
@@ -41,8 +47,20 @@ function CertificationProgramView({
 										versionTested={exam.versionTested}
 										slug={slug}
 									/>
-									<h2 className={s.examAccordionHeading}>
-										{`${title}${examCode ? ` ${examCode}` : ''} Details`}
+
+									<h2
+										className={classNames(
+											s.examAccordionHeading,
+											'g-offset-scroll-margin-top'
```

Comment:
Is this the right way to set the offset? I just copied what was done with the long form content anchor links.

Reply:
This looks right to me!

⛏️ Nitpick, but personally I dig doing `composes: g-offset-scroll-margin-top from global` in the CSS for consistency & to keep the JSX just a lil cleaner... but very nitpicky, seems functional either way so probably the farthest thing from blocking 👍

Reply:
wonderful suggestion, will do that!